edge/services/mms/README.md

```diff
@@ -0,0 +1,149 @@
+
+- Clone this git repo:
+
+```
+git clone git@github.com:open-horizon/examples.git
+```
+
+- Enter the `mms` example directory
+
+```
+cd  examples/edge/services/mms
+```
+
+- Set the values in `horizon/hzn.json` to your own values.
+- Add those values into your shell environment.
+
+```
+`hzn util configconv -f horizon/hzn.json`
+```
+
+- Build and appropriately tag this example Docker container
+
+```
+docker build -t "${DOCKER_IMAGE_BASE}_$(hzn architecture):${SERVICE_VERSION}" .
+```
+
+## Running In Development Mode
+
+- Use the developer tool to run the container with a local development instance of the Cloud Sync Server (CSS). Normally, in production, you will use the CSS in the IBM Cloud, but during development it is convenient to have a dedicated and private "dev CSS" instance you can use. So we will show that approach here first.
+
+```
+hzn dev service start
+```
+
+- Observe the `mms` Service example output (keep this running in a separate terminal so you can watch it as it changes a bit later):
+
+```
+# Soon you will use 'hzn service log ...' for all platforms
+# For now on Linux:
+tail -f /var/log/syslog | grep mms[[]
+# For now on Mac:
+docker logs -f $(docker ps -q --filter name=mms)
+``` 
+
+You should see something similar to this:
+
+```
+Jun  7 16:04:01 myedgenode0 workload-c9ef49dbf715f1477f72c001eb3933970690bea96c4d486a7fc60a686843fcd1_ibm.mms[823]: myedgenode0.dev.edge-fabric.com says: "Hello!"
+Jun  7 16:04:04 myedgenode0 workload-c9ef49dbf715f1477f72c001eb3933970690bea96c4d486a7fc60a686843fcd1_ibm.mms[823]: myedgenode0.dev.edge-fabric.com says: "Hello!"
+...
+```
+
+That is, the output should identify your Edge Node, and the message should be, "**Hello!**". This is how the Service is initially configured. Now let's use the "dev CSS" to send something to the Edge Sync Service (ESS) that the container is monitoring. In a **host**  shell, run this:
+
+```
+echo 'Goodbye!' | ./dev-css-write.sh example-type id-0
+```
+
+- Observe the change in the mms Service example output:
+
+```
+Jun  7 16:04:17 myedgenode0 workload-c9ef49dbf715f1477f72c001eb3933970690bea96c4d486a7fc60a686843fcd1_ibm.mms[823]: myedgenode0.dev.edge-fabric.com says: "Hello!"
+Jun  7 16:04:20 myedgenode0 workload-c9ef49dbf715f1477f72c001eb3933970690bea96c4d486a7fc60a686843fcd1_ibm.mms[823]: myedgenode0.dev.edge-fabric.com says: "Hello!"
+Jun  7 16:04:23 myedgenode0 workload-c9ef49dbf715f1477f72c001eb3933970690bea96c4d486a7fc60a686843fcd1_ibm.mms[823]: myedgenode0.dev.edge-fabric.com says: ""Goodbye!""
+Jun  7 16:04:26 myedgenode0 workload-c9ef49dbf715f1477f72c001eb3933970690bea96c4d486a7fc60a686843fcd1_ibm.mms[823]: myedgenode0.dev.edge-fabric.com says: ""Goodbye!""
+```
+
+- Notice the the message changed to "**Goodbye!**".
+- You can send other messages and watch them being picked up. Just make be to change the object id from `id-0` above to something else each time you send. E.g.:
```

Comment:
You dont have to change the object id. You can just update/change the data that the object id refers to.